keyboards/afternoonlabs/breeze/rev0/info.json

```diff
@@ -6,7 +6,7 @@
     "width": 19,
```

Comment:
Is it necessary to update the rev0?

Reply:
Eh, the build breaks if we don't, and it's relatively easy to do, so /shrug - We don't have to if you don't want to.